library/Ivoz/Cgr/Domain/Service/TpDestinationRate/UpdatedByDestinationRate.php

```diff
@@ -52,6 +53,10 @@ public function execute(DestinationRateInterface $destinationRate)
         $brand = $destinationRate->getDestinationRateGroup()->getBrand();
         $tpDestinationRate = $destinationRate->getTpDestinationRate();
 
+        $roundingMethod = $destinationRate->getDestinationRateGroup()->getDeductibleConnectionFee() ?
```

Comment:
You move this into DestinationRate entity in order to avoid duplicated code on this PR